src/main/java/competition/subsystems/lights/LightSubsystem.java

```diff
@@ -143,6 +170,13 @@ public LightsStateMessage getState() {
         return state;
     }
 
+    public void updateActivity(AlignCameraToAprilTagCalculator.Activity activity) {
+        this.activity = activity;
+    }
+
+    public void updateTargetCameraID(int targetCameraID) {
```

Comment:
⭐  Since you want this information at the same time as activity getting updated, why not combine them into 1 method? That way nobody can forget to provide both at the same time, it's enforced:

```suggestion
public void updateActivity(AlignCameraToAprilTagCalculator.Activity activity, int targetCameraID) {